src/sections/Meshery/meshery.style.js

```diff
@@ -43,7 +43,11 @@ const MesheryWrapper = styled.section`
                      
                     .button-icon{
                         margin-right: 0.625rem;
-                    }    
+                    }   
+                }
+                .align-btn{
+                    bottom: -12px;
+                    background-color: #EBC017;
```

Comment:
@tarunsamanta2k20 Please refer theme to add color.

Reply:
@Chadha93 the button primary is using this color code.

Reply:
Yes it is, I meant to convey -  use theming like this - `${props => props.theme.blackToWhite}; `  to add color, (note this is an example, refer to the theme reference which has the desired color).

Reply:
@Chadha93 got it.